client/diskusage.go

```diff
@@ -67,3 +69,14 @@ type DiskUsageOption interface {
 type DiskUsageInfo struct {
 	Filter []string
 }
+
+type UsageRecordType string
+
+const (
+	UsageRecordTypeInternal    UsageRecordType = "buildkit.internal"
```

Comment:
Do we need `buildkit.` prefix here?

Reply:
Probably not. How about 

```
	UsageRecordTypeInternal    UsageRecordType = "internal"
	UsageRecordTypeFrontend    UsageRecordType = "frontend"
	UsageRecordTypeLocalSource UsageRecordType = "source.local"
	UsageRecordTypeGitCheckout UsageRecordType = "source.git.checkout"
	UsageRecordTypeCacheMount  UsageRecordType = "exec.cachemount"
	UsageRecordTypeRegular     UsageRecordType = "regular"
```

Reply:
+1, also wondering whether we can just use an empty string instead of "regular"

Reply:
Yeah, we don't actually save `UsageRecordTypeRegular` to the metadata by default but it is shown in the output if the value is empty. It is bit weird to show empty type in the output and could think of a better word.

Reply:
Updated